### ISSUE TITLE
3.8.7.2 c11 overpriced??

### ISSUE THREAD

**Issue by ebojali** [RainbowMiner/RainbowMiner#166]
Any issues with c11? Came with the latest update..
![screenshot_20180921-043802_teamviewer](https://user-images.githubusercontent.com/35561346/45857316-68ee0080-bd58-11e8-883b-6a1ad0f9bd17.jpg)

**Comment by ebojali** [comment]
Noticed it might be because of the pool reporting it quite high. 😁 but you might confirm it @RainbowMiner 

![screenshot_20180921-070327_chrome](https://user-images.githubusercontent.com/35561346/45861406-a3619880-bd6c-11e8-8fe8-9a197ab365c5.jpg)

**Comment by RainbowMiner** [comment]
Yes, confirmed. It is a new pattern of overpricing. Yesterday, we had Zergpool doing this fun with HMQ1725. I implemented a more aggressive price protection in the new version. 
I analysed, why this happens: if a pool does not mine one single block for a currency within 24h , the calculations go rogue and the estimated prices rise. Then, of course, all profit switching miners worldwide start to mine this currency, thereby increasing the hashrate dramatically. Which then leads to another increase of the price. So the price goes through the roof.
For now, our price protection will switch to the actual_last24h, for the time that the prices are overdone.
